packages/@tinacms/graphql/src/mdx/index.ts

```diff
@@ -16,7 +16,14 @@ limitations under the License.
 
 */
 
-import { parseMDX, stringifyMDX } from '@tinacms/mdx'
+// import { parseMDX, stringifyMDX } from '@tinacms/mdx'
 
-export { parseMDX }
-export { stringifyMDX }
+// export { parseMDX }
+// export { stringifyMDX }
+// MDX is bundled in a way that Nextjs doesn't like, fake it out for now
```

Comment:
This is a temporary workaround since the MDX bundle seems problematic for Next, so any saves will write this dummy content to all rich-text fields, and reads of rich-text show up empty.